lib/mat8bit.gi

```diff
@@ -111,7 +111,7 @@ InstallOtherMethod( \[\]\:\=,  "For a compressed MatFE",
 #M  <mat> [ <pos1>, <pos2> ] := <val>
 ##
 
-InstallMethod( \[\]\:\=,  "For a compressed MatFE",
+InstallMethod( \[\,\]\:\=,  "For a compressed MatFE",
```

Comment:
write `"For a compressed MatFFE"` (occurs twice)

Reply:
Done.